### PR TITLE
added homework

### PR DESCRIPTION
**Pull request by Yunicorn228**

### CODE REVIEW

File: day7/sheldon/index.css [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+  .mobile__container {
+    display: none;
+  }
+
+  .portrait__container {
+    color: white;
+  }
+
+  .logo {
+    padding: 32px 25px 42px 25px;
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: use padding only on top and left corner, the bottom padding/margin should be determined by the element below

File: day7/sheldon/index.css [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+  }
+
+  .room {
+    font-size: 16px;
+    line-height: 20px;
+    color: #8c99a6;
+    padding-top: 16px;
+  }
+
+  .prograssbar {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: spelling and no bem case

File: day7/sheldon/index.css [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+    display: flex;
+    flex-direction: column;
+  }
+
+  .class {
+    font-size: 24px;
+    line-height: 24px;
+  }
+
+  .room {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: use a better name

File: day7/sheldon/index.css [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+    stroke: #ed882b;
+    stroke-linecap: round;
+  }
+
+  .view__block__title {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .class {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: use a better name

File: day7/sheldon/index.css [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+  }
+
+  .prograssbar__bar {
+    background-color: hsla(0, 0%, 100%, 0.5);
+    width: 100%;
+    height: 4px;
+    margin-top: 15px;
+    border-radius: 2px;
+  }
+  .prograss {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: spelling

File: day7/sheldon/index.css [hunk 6]
```diff
@@ -0,0 +1,410 @@
...
+
+  .prograssbar {
+    width: 300px;
+    padding: 22px 20px 20px 20px;
+    background-color: rgba(255, 255, 255, 0.15);
+    margin-top: 32px;
+    border-radius: 8px;
+  }
+
+  .prograssbar__bar {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: spelling

File: day7/sheldon/index.css [hunk 7]
```diff
@@ -0,0 +1,410 @@
...
+    color: #ea3f58;
+    align-items: center;
+    background-color: #fff;
+    border: 1px solid #f1f2f4;
+    border-radius: 30px;
+    outline: none;
+    box-sizing: border-box;
+  }
+
+  .logOut {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: bem or camel? choose one

File: day7/sheldon/index.css [hunk 8]
```diff
@@ -0,0 +1,410 @@
...
+    font-size: 16px;
+    line-height: 19px;
+    box-sizing: border-box;
+  }
+
+  button svg {
+    padding-left: 18px;
+  }
+
+  .school {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: use a better name

File: day7/sheldon/index.css [hunk 9]
```diff
@@ -0,0 +1,410 @@
...
+  }
+
+  .list__date {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 28px;
+    color: black;
+  }
+
+  .green {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: ? what is this name

File: day7/sheldon/index.css [hunk 10]
```diff
@@ -0,0 +1,410 @@
...
+  }
+
+  .oncomingclass__class {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: black;
+  }
+
+  .oncomingclass__room {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: oncoming and class are 2 words

File: day7/sheldon/index.css [hunk 11]
```diff
@@ -0,0 +1,410 @@
...
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    color: rgba(170, 170, 170, 0.5);
+  }
+
+  .oncomingclass {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: same

File: day7/sheldon/index.css [hunk 12]
```diff
@@ -0,0 +1,410 @@
...
+    padding: 50px 32px 32px 0px;
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: top and bottom need to have same padding

File: day7/sheldon/index.css [hunk 13]
```diff
@@ -0,0 +1,410 @@
...
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .view__block__title {
+    text-align: center;
+  }
+
+  .prograssbar {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: spelling

File: day7/sheldon/index.css [hunk 14]
```diff
@@ -0,0 +1,410 @@
...
+    position: absolute;
+    bottom: 40px;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: white;
+    color: black;
+    width: 80%;
+  }
+
+  .prograssbar__bar {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: same

File: day7/sheldon/index.css [hunk 15]
```diff
@@ -0,0 +1,410 @@
...
+    background-color: #f1f2f4;
+  }
+
+  .prograss {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: same, and use a better name

File: day7/sheldon/index.css [hunk 16]
```diff
@@ -0,0 +1,410 @@
...
+
+  .class {
+    margin-top: 32px;
+    font-size: 24px;
+    line-height: 24px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .room {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: this is the second time i'm seeing class and room here, whats going on?

File: day7/sheldon/index.css [hunk 17]
```diff
@@ -0,0 +1,410 @@
...
+    font-size: 16px;
+    line-height: 20px;
+    color: rgba(255, 255, 255, 0.5);
+    padding-top: 12px;
+    text-align: center;
+  }
+
+  .oncomingclass {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: ?

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]
you are writing alot of the code twice, check your work before submitting

File: day7/sheldon/index.css (hunk 9 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: span that color need to be green

File: day7/sheldon/index.css (hunk 1 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 6 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 7 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 8 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 10 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 11 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed as well

File: day7/sheldon/index.css (hunk 12 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: why

File: day7/sheldon/index.css (hunk 13 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 14 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 15 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 16 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: on different screen boss

File: day7/sheldon/index.css (hunk 17 above)
Comment (Yunicorn228) [reply day7/sheldon/index.css]: fixed

File: day7/sheldon/index.css (hunk 9 above)
Comment (mxdi9i7) [reply day7/sheldon/index.css]: You cannot use green as a class name

File: day7/sheldon/index.css (hunk 12 above)
Comment (mxdi9i7) [reply day7/sheldon/index.css]: Figma says otherwise

File: day7/sheldon/index.css [hunk 18]
```diff
@@ -336,7 +337,7 @@ img {
     text-align: center;
   }
 
-  .oncomingclass {
+  .oncoming-class {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: duplicated code, please fix

File: day7/sheldon/index.css [hunk 19]
```diff
@@ -363,11 +364,11 @@ img {
     padding: 120px 72px 32px 72px;
   }
 
-  .oncomingclass {
+  .oncoming-class {
     margin: 0px 72px;
   }
 
-  .prograssbar {
+  .progress-bar {
```
Comment (mxdi9i7) [inline day7/sheldon/index.css]: same

**Review by mxdi9i7 — APPROVED** [review]

**Pull request merged**